modulo2/projeto-labefy/projeto-labefy/src/App.js

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import GlobalStyle from "./components/App/GlobalStyle";
+
+import Inicio from "./components/Inicio/Inicio";
+import Playlists from "./components/Playlists/Playlists";
+import Explorar from "./components/Explorar/Explorar";
+import DetalhesPlaylist from "./components/Playlists/DetalhesPlaylist";
+import Player from "./components/Player/Player";
+
+import { ContainerPrincipal, ContainerHeader, ContainerLogo, ContainerMenu, ContainerMain, ContainerFooter } from "./components/App/Styles";
+
+class App extends React.Component {
+  state = {
+    paginaAtual: "inicio",
+    idPlaylistDetalhes: "",
+    nomePlaylistDetalhes: "",
+    playerMusica: "",
+    playerArtista: "",
+    playerUrl: "",
+    inicioAtivo: "white",
+    explorarAtivo: "",
+    playlistsAtivo: "",
+    idArrayPlaylist: "",
+  }
+
+  alteraPagina = (e) => {
+    if (e.target.value === "inicio") {
+      this.setState({
+        paginaAtual: e.target.value,
+        inicioAtivo: "white",
+        explorarAtivo: "gray",
+        playlistsAtivo: "gray"
+      })
```

Comment:
Mandou muito bem utilizando o this.setState para alterar vários estados de uma vez! 👍🏿